src/rabbitmanager.js

```diff
@@ -113,6 +113,28 @@ class RabbitManager {
     vhost = encodeURIComponent(vhost);
     await this.request(`permissions/${vhost}/${user}`, {method: 'DELETE'});
   }
+
+  async queues() {
+    return await this.request('queues');
+  }
+
+  async queue(name, vhost='/') {
+    vhost = encodeURIComponent(vhost);
+    return await this.request(`queues/${vhost}/${name}`);
```

Comment:
`vhost` would be URI encoded (percent encoded) on line 122. The default vhost of rabbitmq is '/', and in the URI it appears as "%2f".